README.md

```diff
@@ -75,10 +75,10 @@ Install dependencies for Fedora/CentOS systems:
 yum install make autoconf automake libtool popt-devel xz
 ```
 
-Compilation (`autoreconf` is optional if you obtained source from tarball):
+Compilation (`./autogen.sh` is optional if you obtained source from tarball):
 ```
 cd logrotate-X.Y.Z
-autoreconf -fiv
+./autogen.sh
```

Comment:
What is the advantage of recommending to run ```autogen.sh``` instead of ```autoreconf```?

```autogen.sh``` does not provide any options, neither has its own man page.  When something goes wrong, one would have to debug yet another layer for no apparent benefit.  I thought that ```autogen.sh``` was just a wrapper for compatibility with legacy build scripts.